manifest.json

```diff
@@ -1,9 +1,13 @@
 {
   "name": "Force New Tabs to Main Window",
   "description": "Designate a single main window where new tabs should open",
-  "version": "1.0.4",
+  "version": "1.1.0",
   "manifest_version": 3,
   "permissions": ["tabs", "storage"],
+  "homepage_url": "https://chrome.google.com/webstore/detail/force-new-tabs-to-main-wi/kgojjafnbajjomijiceiefakpphcicen",
```

Comment:
Thanks for setting this!

Reply:
no prob! after hijacking the left click action, it was necessary to add the `homepage_url` so clicking the context menu would still take you to the same place.